frontend/app/routes/protected/person-case/privacy-statement.tsx

```diff
@@ -36,30 +34,43 @@ export async function loader({ context, request }: Route.LoaderArgs) {
   };
 }
 
-export const meta: Route.MetaFunction = ({ data }) => {
+export function meta({ data }: Route.MetaArgs) {
   return [{ title: data.documentTitle }];
-};
+}
 
 export async function action({ context, request }: Route.ActionArgs) {
   requireAuth(context.session, new URL(request.url), ['user']);
   const lang = getLanguage(request);
-
+  const t = await getFixedT(request, handle.i18nNamespace);
   const formData = await request.formData();
+
+  if (formData.get('action') === 'back') {
+    return i18nRedirect('routes/protected/index.tsx', request);
+  }
+
   const input = { confirmPrivacyNotice: formData.get('confirmPrivacyNotice') as string };
 
-  const schema = v.object({ confirmPrivacyNotice: confirmPrivacyNoticeSchema() });
+  const schema = v.object({
+    confirmPrivacyNotice: v.pipe(
+      v.string(t('protected:privacy-statement.confirm-privacy-notice-checkbox.required')),
+      v.trim(),
+      v.nonEmpty(t('protected:privacy-statement.confirm-privacy-notice-checkbox.required')),
+    ),
+  });
   const parsedDataResult = v.safeParse(schema, input, { lang });
 
   if (!parsedDataResult.success) {
     return data({ errors: v.flatten<typeof schema>(parsedDataResult.issues).nested }, { status: 400 });
   }
 
-  // If the first name is valid, store it in the session and redirect to the next page
   context.session.inPersonSINCase = {
     ...(context.session.inPersonSINCase ?? {}),
     ...input,
   };
-  return i18nRedirect('routes/protected/person-case/first-name.tsx', request);
+
+  if (formData.get('action') === 'next') {
+    return i18nRedirect('routes/protected/person-case/first-name.tsx', request); //TODO: change it to redirect to file="routes/protected/person-case/request-details.tsx"
```

Comment:
Use the existing or future route files based of the flow logic.
```suggestion
    return i18nRedirect('routes/protected/person-case/request-details.tsx', request);
```

Reply:
route request-details doesn't exist right now